docs/pages/aws-oss-guide.mdx

```diff
@@ -164,6 +164,7 @@ An example policy is shown below:
                 "s3:GetEncryptionConfiguration",
                 "s3:GetObjectRetention",
                 "s3:ListBucketVersions",
+                "s3:ListBucketMultipartUploads",
```

Comment:
Nit since we are only including that as an **example** policy: does `s3:ListMultipartUploadParts` also solve the issue? https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html#sdksupportformpu

Also, it may be worthwhile to include one or both IAM permissions in an Admonition as a Tip type instead.

Reply:
I was not able to recreate the issue originally myself so cant verify if s3:ListMultipartUploadParts also solves the issue.

Since its for Multipart uploads to S3, it might only occur when a large session recording needs to be uploaded.  I can try to recreate the issue to isolate the conditions that makes it necessary if you think that would add some value to the documentation

Reply:
Can you try `node-sync` mode?

Reply:
@pschisa ^